mlflow/utils/mlflow_tags.py

```diff
@@ -27,6 +27,7 @@
 MLFLOW_DATABRICKS_WEBAPP_URL = "mlflow.databricks.webappURL"
 MLFLOW_DATABRICKS_RUN_URL = "mlflow.databricks.runURL"
 MLFLOW_DATABRICKS_CLUSTER_ID = "mlflow.databricks.cluster.id"
+MLFLOW_DATABRICKS_COMMAND_ID = "mlflow.databricks.commandID"
```

Comment:
Not sure whether to rename this to `"mlflow.databricks.jobGroup.id"`.

Reply:
The current naming seems fine (basically semantically we want this ID to be the unique ID of a command execution in a Databricks notebook). Maybe `mlflow.databricks.notebook.commandId` for clarity? And we could add a clarifying comment to ^that effect explaining what this tag is